Patch status: limbo, but should be fixed when GLX code is removed again

chromium has removed some GLX code which Qt has temporarily(?)
restored in [1]. This however seems to misses some headers and
only works with some configurations and fails with others[2].
So add the headers here as a quick fix for now.

[1] https://github.com/qt/qtwebengine-chromium/commit/f2b407a61bea122d18a012f0049ba193725f0461
[2] https://bugs.gentoo.org/928508
--- a/src/3rdparty/chromium/ui/gl/gl_context.h
+++ b/src/3rdparty/chromium/ui/gl/gl_context.h
@@ -22,4 +22,5 @@
 #include "ui/gl/gl_share_group.h"
 #include "ui/gl/gl_state_restorer.h"
+#include "ui/gl/gl_surface.h"
 #include "ui/gl/gl_workarounds.h"
 #include "ui/gl/gpu_preference.h"
--- a/src/3rdparty/chromium/ui/gl/gl_display.h
+++ b/src/3rdparty/chromium/ui/gl/gl_display.h
@@ -12,4 +12,5 @@
 
 #include "ui/gl/gl_export.h"
+#include "ui/gl/gpu_preference.h"
 
 #if defined(USE_EGL)
